src/doc/snabblab.md

```diff
@@ -0,0 +1,50 @@
+Servers devoted to the Snabb project and usable by all known developers.
+
+Want to be a known developer? Sure! Just edit the [user account list](https://github.com/snabblab/snabblab-nixos/blob/master/modules/users.nix) with your user and send a pull request. No fuss.
+
+## Guidelines
+
+- Feel at home. These servers are here for you to play with and enjoy.
+- Please run Snabb processes like this: `sudo lock ./snabb ...`. The `lock` command will automatically wait if somebody else is running a Snabb process on the same machine and that helps us avoid conflicts for access to hardware resources.
+- Tell `luke@snabb.co` your email address(es) to get an invitation to the [Lab Slack](http://snabb.slack.com/).
+- Don't keep precious data on the servers. We might want to reinstall them at short notice.
+
+## Servers
+
+Name | Purpose | SSH | Xeon model | NICs
+---- | ------- | -------- | -------- | ------ | ----
+lugano-1 | General use | lugano-1.snabb.co | E3 1650v3 | 2 x 10G (82599), 4 x 10G (X710), 2 x 40G (XL710)
+lugano-2 | General use | lugano-1.snabb.co | E3 1650v3 | 2 x 10G (82599), 4 x 10G (X710), 2 x 40G (XL710)
```

Comment:
isn't fqdn in "SSH" column wrong for this and following rows? I guess name and SSH should match up.